packages/ckeditor5-dev-release-tools/lib/tasks/commitandtag.js

```diff
@@ -39,6 +41,19 @@ export default async function commitAndTag( { version, files, cwd = process.cwd(
 		return;
 	}
 
+	if ( dryRun ) {
+		try {
```

Comment:
Could we move the entire try/catch/finally section to a dedicated function?

Reply:
Adressed in [7859810](https://github.com/ckeditor/ckeditor5-dev/pull/1077/commits/7859810ad6c8a1b9659aa2cc2171824b42507622).